### PR TITLE
Sudomaze tutorial

### PR DESCRIPTION
**Pull request by Wikunia**
A subbranch of #88 as I wasn't allowed to add changes to it. 
@sudomaze said we can finish it up @TheCedarPrince

### CODE REVIEW

**Comment by TheCedarPrince** [comment]
@Wikunia I can't tag you as a reviewer, but could you give some reviews here? I had to somewhat overhaul the tutorial as there was a lot to still fix up.

I think I got mostly everything good.

File: docs/src/tutorials/tutorial_1.md [hunk 1]
```diff
@@ -2,6 +2,350 @@
...
+
+By the end of this tutorial, you will have made the following animation:
+
+![](assets/dancing_circles.gif)
+
+With all that said, let's dive in to this tutorial! ✨
+
+## So... What Is `Javis.jl`?
+
+`Javis.jl` is an abstraction on top of powerful graphics tools to make animations and visualizations easy to make.
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:28]: I can't figure out a better way but somehow I thing there must be a way to say it without a double "make" :smile:

File: docs/src/tutorials/tutorial_1.md [hunk 2]
```diff
@@ -2,6 +2,350 @@
...
+It is built on top of the fantastic Julia animation packages, [`Luxor.jl`](https://github.com/JuliaGraphics/Luxor.jl) and [`Cairo.jl`](https://github.com/JuliaGraphics/Cairo.jl).
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:29]: they are mostly drawing packages and not animations packages

File: docs/src/tutorials/tutorial_1.md [hunk 3]
```diff
@@ -2,6 +2,350 @@
...
+> **NOTE:** If you're interested in 2D graphics, you should definitely check out the awesome `Luxor.jl` package.
+> It has a [great tutorial](https://juliagraphics.github.io/Luxor.jl/stable/tutorial/) that will give you an even greater understanding of how `Javis.jl` works.
+
+## The `javis` Function - the Heart of `Javis.jl`
+
+In order to use `Javis`, we will start with the following import:
+
+```julia
+using Javis
+
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:42]: I would remove this empty line

File: docs/src/tutorials/tutorial_1.md [hunk 4]
```diff
@@ -2,6 +2,350 @@
...
+### `video`
+
+The `video` argument requires a `Video` struct.
+This `Video` struct creates the canvas upon which we will draw. 🎨 
+To create a `Video` object, one defines the constructor with a width and height:
+
+```julia
+myvideo = Video(500,500) # 500 x 500 // width x height
+
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:71]: same here: I would remove the empty line

File: docs/src/tutorials/tutorial_1.md [hunk 5]
```diff
@@ -2,6 +2,350 @@
...
+```
+
+### `actions`
+
+Core to not only the `javis` function but the entire `Javis.jl` library are `Action` objects. 
+`Action` objects will be better explained in [Tutorial 2](tutorial_2.html).
+For now, just know that an `Action` object is what draws shapes or moves shapes on a `Video`. In the `javis` function, the `actions` kwarg is a vector of `Action` objects.
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:78]: it is not actually a `kwarg` but I know you love `kwarg` :heart:

File: docs/src/tutorials/tutorial_1.md [hunk 6]
```diff
@@ -2,6 +2,350 @@
...
+> These arguments are defined below:
+>
+> - `video`: Video struct
+> - `action`: Action struct
+> - `frame`: the current frame number
+>
+> Although these arguments are irrelevant for the `ground` function, we need to write `args...` such that Julia actually knows that we have a method that accepts those three arguments.
+> The `...` basically stands for as many arguments as you want.
+
+As `Javis` does not provide a method for drawing circles, we need to make one ourselves!
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:119]: I would rephrase as Luxor provides the functionality to draw circles but we also want to specify the color of the circle here.

File: docs/src/tutorials/tutorial_1.md [hunk 7]
```diff
@@ -2,6 +2,350 @@
...
+```
+
+![](assets/multiple_circles.gif)
+
+So, this drawing - it's all nice and all, but perhaps a little...
+Dull? 
+This is supposed to be an animation!
+Let's make these balls dance. 💃
+
+## It Takes Two to Tango 💃
```
Comment (Wikunia) [inline docs/src/tutorials/tutorial_1.md:173]: Love it!

File: docs/src/tutorials/tutorial_1.md (hunk 5 above)
Comment (TheCedarPrince) [reply docs/src/tutorials/tutorial_1.md:78]: image
